numba_dppy/config.py

```diff
@@ -47,10 +47,11 @@ class DpctlMinimumVersionRequiredError(Exception):
         msg += "A non-host SYCL device is required to use numba_dppy."
         warnings.warn(msg, UserWarning)
 except DpctlMinimumVersionRequiredError:
-    msg = "numba_dppy requires dpctl 0.7.0 or higher."
+    msg = "numba_dppy is not compatible with " + dpctl.__version__ + "."
+    msg += " Install dpctl 0.8.* or higher."
     warnings.warn(msg, UserWarning)
 except:
-    msg = "Please install dpctl 0.7.0 or higher."
+    msg = "Please install dpctl 0.8.* or higher."
```

Comment:
This file contains 5 places with dpctl version. All should be updated next time.
I propose creating function like: `dpctl_required("0.8.*")` which will throw exception is dpctl version does not match.